sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/sharedconfig/common.webpack.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  module: {
+    loaders: [
+     {
+        test: /\.(jpe?g|png|gif|svg)$/i,
+        loaders: [
+          'file?hash=sha512&digest=hex&name=[hash].[ext]',
+          'image-webpack?bypassOnDebug&optimizationLevel=7&interlaced=false'
+        ]
+      }
+    ],
```

Comment:
According to the [CI (line 1170)](https://ci.appveyor.com/project/julienrf/scalajs-bundler/build/217#L1170) it seems that at some point a `require('fs')` is performed *from the browser*… Do you know why? Do the tests run fine on your machine?